client/state/rewind/size/test/reducer.js

```diff
@@ -82,4 +82,30 @@ describe( 'rewind.size reducers', () => {
 			).toEqual( 7 );
 		} );
 	} );
+	describe( 'backupsStopped', () => {
```

Comment:
Sorry for nit, could we add an additional line break before the `describe` here?
```suggestion

	describe( 'backupsStopped', () => {
```